newIDE/electron-app/app/main.js

```diff
@@ -140,11 +140,12 @@ app.on("ready", function() {
 
   ipcMain.on(
     "piskel-changes-saved",
-    (event, imageResources, newAnimationName) => {
+    (event, imageResources, newAnimationName, fileMetadata) => {
       mainWindow.webContents.send(
         "piskel-changes-saved",
         imageResources,
-        newAnimationName
+        newAnimationName,
+        fileMetadata
```

Comment:
This should be named "externalEditorData" if we're consistent right? 

I know that I'm a pain with naming things, but if we don't strive for simplicity and proper naming, we'll end up with 3 different naming for a single concept, and everything will look super complicated when going back to this code in 2 months :)

Reply:
A proper naming will "flow" though the code without being changed. If the name of the variable changes, this means that what is represents is different.

Reply:
can I call it 'receivedData' here? I am asking because after it is passed on, it is actually them put in a constant with the name 'externalEditorData'
we can then have:
```
      const externalEditorData = {
        pskl: receivedData,
      }
```
that might be a bit confusing

Reply:
receivedData is fine.

Also, if I take a look at:

```
      const externalEditorData = {
        pskl: receivedData,
      }
```

this is actually metadata right?

Reply:
yes its metadata. If you look at the function, if I do this:
```
      const externalEditorData = {
        pskl: receivedData,
      }
```
externalEditorData  will be the name of the variable that gets passed to onChangesSaved- as you requested earlier on the flow note :)